content/scalers/mysql.md

```diff
@@ -0,0 +1,101 @@
++++
+fragment = "content"
+weight = 100
+title = "MySQL"
+background = "light"
++++
+
+Scale applications based on MySQL query result.
+
+**Availability:** v1.1+ | **Maintainer:** Community
+
+<!--more-->
+
+### Trigger Specification
+
+This specification describes the `mysql` trigger that scales based on result of MySQL query.
+The trigger requires the following information:
+- `query` - a MySQL query that should return single numeric value that will be used for HPA
+- `queryValue` - a threshold that is used as `targetAverageValue` in HPA. 
+- Connection information as `dsnConnectionStr` or set of required params (more information below)
+
+#### Using `dsnConnectionStr`
+One option to connect to MySQL database is to use dsn connection string from environment variable:
+```yaml
+triggers:
+  - type: mysql
+    metadata:
+      dsnConnectionStr: "SQL_CONN_STR" // SQL_CONN_STR env variable should point to valid connection string
```

Comment:
I would make it so that `dsnConnectionStr` can be configured via `TriggerAuthentication` as well

Reply:
It will work also with `TriggerAuthentication`